### PR TITLE
Feature/custom code: menu & promotion.

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: includes/settings/settings.php [hunk 1]
```diff
@@ -149,6 +149,16 @@ public function register_pro_menu() {
...
 
+		add_submenu_page(
+			self::PAGE_ID,
+			__( 'Custom Code', 'elementor' ),
+			__( 'Custom Code', 'elementor' ),
+			'manage_options',
+			'elementor_custom_custom_code',
+			[ $this, 'elementor_custom_code' ]
+		);
+
```
Comment (rodolphebertozzo) [inline includes/settings/settings.php:160]: Remove line

Comment (matipojo) [reply includes/settings/settings.php:160]: Why?

**Review by matipojo — APPROVED** [review]

File: includes/settings/settings.php [hunk 2]
```diff
@@ -149,6 +149,15 @@ public function register_pro_menu() {
 			[ $this, 'elementor_custom_icons' ]
 		);
 
+		add_submenu_page(
+			self::PAGE_ID,
+			__( 'Custom Code', 'elementor' ),
+			__( 'Custom Code', 'elementor' ),
+			'manage_options',
+			'elementor_custom_custom_code',
+			[ $this, 'elementor_custom_code' ]
```
Comment (kobizz) [inline includes/settings/settings.php:158]: Use an anonymous callback with reference to a private method

**Review by kobizz — CHANGES_REQUESTED** [review]

**Review by kobizz — APPROVED** [review]

**Review by rodolphebertozzo — APPROVED** [review]

File: includes/settings/settings.php [hunk 3]
```diff
@@ -149,6 +149,17 @@ public function register_pro_menu() {
 			[ $this, 'elementor_custom_icons' ]
 		);
 
+		add_submenu_page(
+			self::PAGE_ID,
+			__( 'Custom Code', 'elementor' ),
+			__( 'Custom Code', 'elementor' ),
+			'manage_options',
+			'elementor_custom_custom_code',
+			function () {
```
Comment (KingYes) [inline includes/settings/settings.php:158]:
```suggestion
			function() {
```

**Review by KingYes — CHANGES_REQUESTED** [review]

**Pull request merged**